libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestRequest+Internal.h

```diff
@@ -26,7 +26,9 @@
 
 @interface SFRestRequest ()
 
+@property (nonnull, nonatomic, strong, readwrite) NSMutableURLRequest *request;
```

Comment:
We should be consistent - since we started using NS_ASSUME_NONNULL_BEGIN in some places, maybe we should start putting it in any headers we work on.